scan_to_inventory/static/www/models/m_stock_inventory.js

```diff
@@ -0,0 +1,39 @@
+angular.module('scan_to_inventory').factory(
+        'StockInventoryModel', [
+        '$q', '$rootScope', 'jsonRpc',
+        function ($q, $rootScope, jsonRpc) {
+
+    return {
+        LoadDraftInventoryList: function() {
+            return jsonRpc.searchRead(
+                    'stock.inventory', [['state', '=', 'draft']], [
+                    'id', 'name', 'date', 'inventory_line_qty',
+                    ]).then(function (res) {
+                $rootScope.DraftInventoryList = res.records;
+                return res.records.length;
+            });
+        },
+        CreateInventory: function(name) {
+            return jsonRpc.call(
+                    'stock.inventory', 'create_by_scan',
+                    [name]).then(function (res) {
```

Comment:
I did not understood this part. but I guess that the first return is useless. (return jsonRpc.call).
The interest of this function is to store inventory quantity in a global var too, so we have to wait, don't we ?